Controller/src/LSST/M1M3/SS/Accelerometer/Accelerometer.h

```diff
@@ -15,35 +15,25 @@ namespace LSST {
 namespace M1M3 {
 namespace SS {
 
-class AccelerometerSettings;
 struct SupportFPGAData;
+class AccelerometerSettings;
 class M1M3SSPublisher;
 
 /*!
  * The class used to process accelerometer data.
  */
 class Accelerometer {
-   private:
-    AccelerometerSettings* accelerometerSettings;
-    SupportFPGAData* fpgaData;
-    M1M3SSPublisher* publisher;
-
-    MTM1M3_accelerometerDataC* accelerometerData;
-    MTM1M3_logevent_accelerometerWarningC* accelerometerWarning;
-
    public:
-    /*!
-	 * Instantiates the accelerometer.
-	 * @param[in] accelerometerSettings The accelerometer settings.
-	 * @param[in] fpgaData The fpga data.
-	 * @param[in] publisher The publisher.
-	 */
     Accelerometer(AccelerometerSettings* accelerometerSettings, SupportFPGAData* fpgaData, M1M3SSPublisher* publisher);
-
-    /*!
-	 * Processes currently available accelerometer data and publish it.
-	 */
+    ~Accelerometer() = default;
     void processData();
+
+   private:
+    AccelerometerSettings* accelerometerSettings = nullptr;
+    SupportFPGAData* fpgaData = nullptr;
+    M1M3SSPublisher* publisher = nullptr;
+    MTM1M3_accelerometerDataC* accelerometerData = nullptr;
+    MTM1M3_logevent_accelerometerWarningC* accelerometerWarning = nullptr;
```

Comment:
Please consider using `std::shared_ptr` or `std::unique_ptr` instead of bare pointers here, and everywhere else it is practical. That should make the code more robust and allow use of default destructors.

That is almost certainly out of scope for this ticket, but if so, would be worth a new ticket.